### ISSUE TITLE
DropDownTree accessibility issues

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6620]
- [AccessibilityBug] **DropDownTree** should have `aria-owns` attribute on its wrapper element pointing to its popup element; - https://github.com/telerik/kendo-ui-core/issues/6558
- [x] Resolve all AXE test violations;
- [x] Screen readers do not announce selected item upon focus;
